Bug: 无法在qq适配器工作

### 描述问题

在qq适配器插件会报错

### 复现步骤

测试平台: adapter-qq
插件版本: 2.1.1


### 期望的结果

_No response_

### 截图或日志

```log
11-10 22:54:57 [ERROR] nonebot | Running Matcher(type='message', module=plugins.nonebot_plugin_parser.matchers, lineno=38) failed.
Traceback (most recent call last):
  File "E:\QBot\bot.py", line 15, in <module>
    nonebot.run()
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\__init__.py", line 337, in run
    get_driver().run(*args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\drivers\none.py", line 61, in run
    anyio.run(self._serve)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\anyio\_core\_eventloop.py", line 74, in run
    return async_backend.run(func, args, {}, backend_options)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\anyio\_backends\_asyncio.py", line 2316, in ru
n
    return runner.run(wrapper())
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 641, in run_until_complete
    self.run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 608, in run_forever
    self._run_once()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 1936, in _run_once
    handle._run()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\asyncio\events.py", line 84, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\utils.py", line 259, in run_coro_with_
shield
    return await coro
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\message.py", line 508, in check_and_ru
n_matcher
    await _run_matcher(
> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\message.py", line 460, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\internal\matcher\matcher.py", line 926
, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\internal\matcher\matcher.py", line 863
, in simple_run
    await handler(
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot\dependencies\__init__.py", line 113, i
n __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "E:\QBot\plugins\nonebot_plugin_parser\matchers\__init__.py", line 56, in parser_handler
    await _message_reaction(event, "resolving")
  File "E:\QBot\plugins\nonebot_plugin_parser\matchers\__init__.py", line 111, in _message_reaction
    await uniseg.message_reaction(emoji, message_id=message_id)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot_plugin_alconna\uniseg\functions.py", l
ine 94, in message_reaction
    return await fn.reaction(emj, message_id or fn.get_message_id(event), bot, event, delete=delete)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-oFP0POfx-py3.11\Lib\site-packages\nonebot_plugin_alconna\uniseg\adapters\qq\expo
rter.py", line 464, in reaction
    assert emoji.id.isdigit()
AssertionError

```

可能是以下代码导致，建议增加对 `adapter-qq` 的判断
```py
# mathers/__init__.py
106     if target.adapter == SupportAdapter.onebot11:
107         emoji = emoji_map[status][0]
108     else:
109        emoji = emoji_map[status][1]
```

## 修复意见
```py
# mathers/__init__.py
106     if target.adapter in [SupportAdapter.onebot11, SupportAdapter.qq]:
107         emoji = emoji_map[status][0]
108     else:
109        emoji = emoji_map[status][1] 
```

--- a/src/nonebot_plugin_parser/matchers/__init__.py
+++ b/src/nonebot_plugin_parser/matchers/__init__.py
@@ -98,18 +98,22 @@ async def _message_reaction(
     status: Literal["fail", "resolving", "done"],
 ) -> None:
     emoji_map = {
-        "fail": ["10060", "❌"],
-        "resolving": ["424", "👀"],
-        "done": ["144", "🎉"],
+        "fail": ("10060", "❌"),
+        "resolving": ("424", "👀"),
+        "done": ("144", "🎉"),
     }
     message_id = uniseg.get_message_id(event)
     target = uniseg.get_target(event)
-    if target.adapter == SupportAdapter.onebot11:
+
+    if target.adapter in (SupportAdapter.onebot11, SupportAdapter.qq):
         emoji = emoji_map[status][0]
     else:
         emoji = emoji_map[status][1]
 
-    await uniseg.message_reaction(emoji, message_id=message_id)
+    try:
+        await uniseg.message_reaction(emoji, message_id=message_id)
+    except Exception:
+        logger.warning(f"reaction {emoji} to {message_id} failed, maybe not support")
 
 
 import re

